storageFile.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
move all csv files to `resources` folder (both for `main` and `test`)

Reply:
fixed